### PR TITLE
Stemming fixes, finally

### PR DESCRIPTION
**Pull request by Firesphere**

### CODE REVIEW

File: Solr/5/templates/schema.ss [hunk 1]
```diff
@@ -32,9 +32,9 @@
  PERFORMANCE NOTE: this schema includes many optional features and should not
  be used for benchmarking.  To improve performance one could
   - set stored="false" for all fields possible (esp large fields) when you
-    only need to search on the field but don't need to return the original
+    only need to search on the field but don"t need to return the original
```
Comment (marczhermo) [inline Solr/5/templates/schema.ss]:
```suggestion
    only need to search on the field but don't need to return the original
```

File: Solr/5/templates/schema.ss [hunk 2]
```diff
@@ -32,9 +32,9 @@
...
     value.
-  - set indexed="false" if you don't need to search on the field, but only
+  - set indexed="false" if you don"t need to search on the field, but only
```
Comment (marczhermo) [inline Solr/5/templates/schema.ss]:
```suggestion
  - set indexed="false" if you don't need to search on the field, but only
```

File: Solr/5/templates/schema.ss (hunk 1 above)
Comment (Firesphere) [reply Solr/5/templates/schema.ss]: I think I've been a bit too trigger-happy with the find-and-replace :laughing:

**Comment by Firesphere** [comment]
Ignoring CodeCov, I think this is good to go now.

**Comment by Firesphere** [comment]
I think this is good to go now

**Pull request merged**